src/components/TrackTile/hooks/useAxiosTrackTileService.ts

```diff
@@ -41,10 +41,8 @@ export const useAxiosTrackTileService = (): TrackTileService => {
   const { activeSubjectId: patientId, activeProject } = useActiveProject();
   const { account } = useActiveAccount();
 
-  const accountSettings = {
-    account: account?.id ?? '',
-    project: activeProject?.id ?? '',
-  };
```

Comment:
However, I'm realizing the issue is pushed back up to here.  Not sure if we need to do anything though because we should be protected by isLoading at this point.

This is also in track-tile-ui [here](https://github.com/lifeomic/life-research-mobile/blob/master/src/hooks/useTrackTileService.tsx#L17).